### ISSUE TITLE
Feature Request: Drop shadows

### ISSUE THREAD

**Issue by cskoghed** [leodevbro/vscode-blockman#131]
This extension is amazing. I really wish I could have the possibility to add drop shadows to code blocks though. Either way - great job so far!

**Comment by leodevbro** [comment]
Thanks so much. Well, I'll try to implement drop shadows. The thing is that probably it is not a trivial task, because Blockman does not render block as entire div box, but it renders div boxes on each line, and these line-boxes touch each other and create an illusion of one big box. For simple cases, drop shadow maybe can be easy to implement, but for edge cases (like non-rectangle blocks because of code structure with brackets), it will require some trickery-tricks.

**Comment by cskoghed** [comment]
Yeah, I noticed that when trying to create a hacky solution using
radiant-gradients on top and bottom. Perhaps only allowing shadows for
indent detected blocks would help? But then that feature would kind of need
to be extended to more languages anyways which I assume is not there for
the same reason. Either way, king for replying so fast!

mån 5 feb. 2024 kl. 23:55 skrev Levan Katsadze ***@***.***>:
